docs/data/introduction/installation/installation.md

```diff
@@ -0,0 +1,22 @@
+# Installation
+
+<p class="description">Install the necessary packages to start building with MUI X components.</p>
+
+MUI X packages are available through the free MIT-licensed Community Plan, or the commercially-licensed Pro and Premium Plans.
+See [the licensing page](/x/introduction/licensing/) for complete details.
+
+## Peer dependencies
+
+MUI X components have a peer dependency on `@mui/material`: the installation [instructions](/material-ui/getting-started/installation/).
+
+## Components
+
+Note that you only need to install the packages corresponding to the components you're using—e.g. data grid users don't need to install date and time pickers.
```

Comment:
Should it be `:::info` instead of a sentence starting by "Note"

We tend to replace them in the doc
But here the note is the full section so maybe not

Reply:
That was my thought as well—no need for the callout since it's the entire section.